source/common/config/utility.h

```diff
@@ -269,6 +268,30 @@ class Utility {
     return config;
   }
 
+  /**
+   * Translate the typed any field into a proto message provided by the implementation factory.
+   * @param typed_config typed configuration.
+   * @param validation_visitor message validation visitor instance.
+   * @param factory implementation factory with the method 'createEmptyConfigProto' to produce a
+   * proto to be filled with the translated configuration.
+   */
+  template <class Factory>
+  static ProtobufTypes::MessagePtr
+  translateAnyToFactoryConfig(const ProtobufWkt::Any& typed_config,
```

Comment:
Why is this a standalone change disconnected from the API the rest of implementatinon? I suggest this be a distinct PR or merged with implementation, it doesn't belong in an API only PR.

Reply:
Remove the utility change